.golangci.yml

```diff
@@ -40,7 +40,7 @@ linters:
     # - gofumpt
     - goheader
     - goimports
-    - gomoddirectives
+    # - gomoddirectives
```

Comment:
The linter is disabled as it raises a package replacement warning, which is required to compile KH successfully.